Class/C-Array-Methods/exercise3.js

```diff
@@ -12,8 +12,14 @@ let doubledNumbers = [];
 // Write a "for" loop to ITERATE through the numbersList array
 // Call the function doubleValue for each number and ADD (push) the result to the doubledNumbers array
 
+for (let i = 0; < x; * i++ ){
+
+    if (doubledNumbers[i] <=== 12){
+        result.push(doubledNumbers[i]);

```

Comment:
You don't need to use `if` here - instead you should be applying `doubleNumbers` (the function on line 6) to the `numbersList` element corresponding to that iteration of the loop and pushing it on to the `doubledNumbers` array.

As an aside, the comparison operator you have used in the `if` conditional isn't valid, it should be `<=` if you wanted to do 'less than or equal to'. 

There is a guide to them here:

https://www.w3schools.com/js/js_comparisons.asp